bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -109,6 +109,9 @@ def exit_cleaning():
             pass
 atexit.register(exit_cleaning)
 
+# Which templates might have triggers, initially we'll assume they all might
```

Comment:
The assignment of 'None' here is a bit confusing in the light of 'initially we'll assume they all might' .. suggest eg 'If some templates have no triggers, tids_with_trigs will contain only the ones that do'

Reply:
Ok, done